### PR TITLE
ETCD storage DatetimeField

### PR DESCRIPTION
**Pull request by atengler**
This PR adds DatetimeField class to ETCD storage module. DatetimeField stores datetimes as UTC timestamps (floating point numbers) on the backend and returns them as naïve datetime.

### CODE REVIEW

File: kqueen/config/base.py [hunk 1]
```diff
@@ -8,6 +8,10 @@ class BaseConfig:
     KQUEEN_HOST = '127.0.0.1'
     KQUEEN_PORT = 5000
 
+    # Babel default settings
+    BABEL_DEFAULT_TIMEZONE = 'UTC'
```
Comment (tomkukral) [inline kqueen/config/base.py]: We should probably keep backend on UTC everytime.

File: kqueen/models.py [hunk 2]
```diff
@@ -30,6 +31,8 @@ class Cluster(Model, metaclass=ModelMeta):
     state = StringField()
     kubeconfig = JSONField()
     metadata = JSONField()
+    created_at = DatetimeField()
+    deleted_at = DatetimeField()
```
Comment (tomkukral) [inline kqueen/models.py]: Do you plan set `deleted_at` instead of deleting objects? It won't make sense IMO.

File: kqueen/server.py [hunk 3]
```diff
@@ -1,4 +1,5 @@
 from flask import Flask
+from flask.ext.babel import Babel
```
Comment (tomkukral) [inline kqueen/server.py]: Can we do same with `python-dateutil` and `pytz`?

File: requirements.txt [hunk 4]
```diff
@@ -1,4 +1,5 @@
 Flask==0.12.2
+Flask-Babel=0.11.2
```
Comment (tomkukral) [inline requirements.txt]: Should be `Flask-Babel==0.11.2`

File: setup.py [hunk 5]
```diff
@@ -21,6 +21,7 @@
     zip_safe=False,
     install_requires=[
         'Flask==0.12.2',
+        'Flask-Babel=0.11.2',
```
Comment (tomkukral) [inline setup.py]: Should be `Flask-Babel==0.11.2`

File: kqueen/models.py (hunk 2 above)
Comment (atengler) [reply kqueen/models.py]: I will remove deleted_at field

File: kqueen/server.py (hunk 3 above)
Comment (atengler) [reply kqueen/server.py]: Not with the same functionality. But if you want to keep backend on UTC always, there has to be date parsing and transformation in API client in the UI, so I might as well only return the UTC timestamp and move the smart stuff to UI.

**Pull request merged**